modules/hcp-import-limitations.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-import.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-import-limitations_{context}"]
+= Limitations of managing imported hosted clusters
+
+Hosted clusters are automatically imported into the local {mce}, unlike a standalone {product-title} or third party clusters. Hosted clusters run some of their agents in the _hosted mode_ so that the agents do not use the resources of your cluster. If you choose to automatically import hosted clusters, you can update hosted clusters by using the `HostedCluster` resource on the management cluster.
+
```

Comment:
How? and what in the HostedCluster should I change/configure?

Reply:
What/how to update:[ node pools](https://docs.openshift.com/container-platform/4.17/hosted_control_planes/hcp-updating.html#hcp-update-node-pools_hcp-updating ) and [control plane ](https://docs.openshift.com/container-platform/4.17/hosted_control_planes/hcp-updating.html#hcp-update-ocp-hc_hcp-updating)